.circleci/config.yml

```diff
@@ -133,12 +133,16 @@ references:
   upload_build_artifacts: &upload_build_artifacts
     run:
       name: Upload build artifacts to central repository (AWS)
-      command: aws s3 cp ${PACKAGE_TARBALL:?} s3://${S3_BUILDS_BUCKET:?}/
+      command: |
+        aws s3 cp ${PACKAGE_TARBALL:?} s3://${S3_BUILDS_BUCKET:?}/
+        aws s3api put-object-acl --acl public-read --bucket ${S3_BUILDS_BUCKET:?} --key `basename ${PACKAGE_TARBALL:?}`
```

Comment:
Is this line restricting or relaxing permissions?

Setting ACL after having uploaded artefact seems strange to me, as execution of this script may terminate in between commands, leaving package uploaded but in unexpected state. Why not using [`cp` option `--acl`](https://docs.aws.amazon.com/cli/latest/reference/s3/cp.html#options)?

Reply:
It's relaxing.

However, good point about `cp --acl`. @acidsennui Any reason it's separate API call?